lib/subscription.js

```diff
@@ -18,31 +18,34 @@ const rclnodejs = require('bindings')('rclnodejs');
 const generator = require('../rosidl_gen/generator.js');
 
 class Subscription {
-  constructor(handle, messageType, message, topic, callback) {
+  constructor(handle, messageType, topic, callback) {
     this._handle = handle;
-    this._messageType = messageType,
     this._topic = topic;
     this._callback = callback;
-    this._Message = message;
-    this._rosMsg = new message();
+    this._Message = messageType.Message;
```

Comment:
That's right, I will make this change when the message generator make the corresponding change.